ember-cli-build.js

```diff
@@ -29,5 +29,11 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
+  app.import('bower_components/blueimp-file-upload/js/vendor/jquery.ui.widget.js');
+  app.import('bower_components/blueimp-file-upload/js/jquery.iframe-transport.js');
+  app.import('bower_components/cloudinary/js/load-image.min.js');
+  app.import('bower_components/cloudinary/js/canvas-to-blob.min.js');
+  app.import('bower_components/cloudinary/js/jquery.cloudinary.js');
+
```

Comment:
yeah @mcm-ham 
Also we want to display some transformations to images, like on browse page, we will be showing the main categories images in circular format. For those transformation we will need other js right? (am not much sure about which js files it would need exactly)